plugins/uefi-dbx/fu-plugin-uefi-dbx.c

```diff
@@ -7,18 +7,19 @@
 #include "config.h"
 
 #include "fu-plugin-vfuncs.h"
-#include "fu-efivar.h"
+//#include "fu-efivar.h"
 #include "fu-hash.h"
-#include "fu-efi-signature-common.h"
-#include "fu-efi-signature-parser.h"
-#include "fu-uefi-dbx-common.h"
+//#include "fu-efi-signature-common.h"
+#include "fu-efi-signature-list.h"
+//#include "fu-uefi-dbx-common.h"
```

Comment:
don't forget to remove these commented out things

Reply:
Oops, thanks!